src/mono/wasm/debugger/DebuggerTestSuite/Inspector.cs

```diff
@@ -194,14 +206,28 @@ void FailAllWaiters(Exception? exception = null)
             return ($"console.{type}: {output}", type);
         }
 
-        async Task OnMessage(string method, JObject args, CancellationToken token)
+        async Task OnMessage(string sessionId, string method, JObject args, CancellationToken token)
         {
             bool fail = false;
             switch (method)
             {
+                case "Target.attachedToTarget":
+                {
+                    var sessionIdNewTarget = new SessionId(args["sessionId"]?.Value<string>());
+                    await Client.SendCommand(sessionIdNewTarget, "Profiler.enable", null, token);
+                    await Client.SendCommand(sessionIdNewTarget, "Runtime.enable", null, token);
+                    await Client.SendCommand(sessionIdNewTarget, "Debugger.enable", null, token);
+                    await Client.SendCommand(sessionIdNewTarget, "Runtime.runIfWaitingForDebugger", null, token);
+                    await Client.SendCommand(sessionIdNewTarget, "Debugger.setAsyncCallStackDepth", JObject.FromObject(new { maxDepth = 32}), token);
```

Comment:
For a follow up PR - the commands here, and in `DebuggerTestBase.InitializeAsync` should use the same base set of commands. And each case can add any additional ones as needed. Like the ones here seem to be the same.